controllers/workspace/devworkspace_controller.go

```diff
@@ -121,8 +122,19 @@ func (r *DevWorkspaceReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
+
+	config, err := wkspConfig.ResolveConfigForWorkspace(rawWorkspace, clusterAPI.Client)
+	if err != nil {
+		reqLogger.Error(err, "Error applying external DevWorkspace-Operator configuration")
```

Comment:
We should reserve error logs for issues with the Operator itself; issues encountered here would be due to an issue in how the workspace is defined and so should be logged at info level with a warning:
```suggestion
		reqLogger.Info(fmt.Sprintf("Failed to apply external DevWorkspace-Operator configuration: %s", err)
```

As a future TODO, this should also probably set a warning condition on the workspace, but that's a task for another day.